tests/accuracy/test_global_correction.py

```diff
@@ -195,3 +245,59 @@ def _create_fake_measurement(tmp_path, model_path, twiss_path, error_val, optics
             meas[ERROR] = meas.loc[:, f"{ERR}{col}"].to_numpy()
         meas[WEIGHT] = 1.
     return twiss_df, model_df, meas_dict
+
+# Is the following better placed in a folder with reference files, 
```

Comment:
I think better to create the twiss_dpp files beforehand and simply load. 
Especially as we want to get rid of the sequence files in our repo eventually anyway.
Maybe use the acc-models-lhc way and put the creation in a script next to the twiss_dpp output files for reference and easy future recreation.